assets/drupal-rector.php

```diff
@@ -11,6 +11,7 @@
 use DrupalRector\Set\Drupal8SetList;
 use DrupalRector\Set\Drupal9SetList;
 use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
 use Rector\Set\ValueObject\SetList;
```

Comment:
@jfauske It makes the following import obsolete, right?

Reply:
I know a workflow that would have fixed that, and the D8 and 9 ones 😅 Will fix, but not today

Reply:
Fixed in 97b743f